app/components/navigation/navigation.module.scss

```diff
@@ -16,22 +16,32 @@
     background-color: colors.$blue;
     width: 100%;
     @media (min-width: 768px){
-        width: 10rem;
+        width: 15rem;
     }
     
 }
 
 .top-level-link {
-    padding: 0.5rem;
-    &::after {
-        content: '+';
-        position: absolute;
-        right: 0.5rem;
-    }
+    padding: 0.75rem;
+    line-height: 1rem;
+    position: relative;
+    text-wrap: nowrap;
+    text-overflow: ellipsis;
+    overflow: hidden;
+    padding-right: 1.75rem;
+    width: 100%;
+}
+.nav-dropdown::after {
+    content: '+';
+    display: block;
```

Comment:
noob: what's this for?

Reply:
This is the + sign in the navigation drop down

Reply:
Not the + sign. I am asking about `display: block`

Reply:
Ahh, the `display:block` is to allow the element to be styled as if it were a div. MDN's definition: "The element generates a block box, generating line breaks both before and after the element when in the normal flow."

Here's more info: https://developer.mozilla.org/en-US/docs/Web/CSS/display

Reply:
I guess, what does that mean for us here? Is there a practical reason we need this, or is this more of a "this is the proper way for us to style this so that we don't shoot ourselves in the foot later"?